src/preprocess/index.ts

```diff
@@ -32,6 +32,7 @@ import {_getParamsByPath} from '../params';
 import {Visitor} from './visitor';
 
 handlebars.registerHelper('tojson', (context: any) => JSON.stringify(context));
+handlebars.registerHelper('tojsonPretty', (context: any) => JSON.stringify(context, null, ' '));
```

Comment:
I think this would be more readable as a number instead of a string.
Also, is 1 space enough? I don't know what JSON typically is which would ease people's readability, but I usually use 4 spaces and tools like `jq` usually use 2 spaces. Given not many people would likely look at it too often, it shouldn't be a large issue either way.

Reply:
I went for 1 due to the size limits on cfn templates not coming form s3. The most important thing is the new lines it inserts.